lighthouse-core/report/html/renderer/pwa-category-renderer.js

```diff
@@ -18,6 +18,16 @@
 
 /* globals self, Util, CategoryRenderer */
 
+/**
+ * An always-increasing counter for making unique SVG ID suffixes.
+ */
+const svgSuffixCounter = (() => {
+  let svgSuffix = 0;
+  return function() {
+    return svgSuffix++;
+  };
```

Comment:
this function is not my favorite thing but felt a little bit better than `Math.floor(Math.random() * 9999)` as a suffix generator. I can switch and avoid this if anyone feels strongly, though.

Reply:
`getUniqueSuffix` 👍